tests/test_caching.py

```diff
@@ -0,0 +1,160 @@
+import logging
+
+import pandas as pd
+import pytest
+
+from hamilton import base
+from hamilton.driver import Driver
+from hamilton.experimental.h_cache import (
```

Comment:
this is a nit pick, but I think module imports are nicer --  then when you read you know where the function definition lives.  But happy to be convinced otherwise.
```suggestion
from hamilton.experimental import h_cache
```

Reply:
Yeah, I actually agree, often I do it myself but as you can see, I struggle with consistency :)